tests/pillar/repo_mos9.sls

```diff
@@ -0,0 +1,8 @@
+linux:
+  system:
+    enabled: true
+    repo:
+      mirantis_openstack:
+        source: "deb [arch=amd64] http://mirror.fuel-infra.org/mos-repos/ubuntu/9.0/ mos9.0 main restricted"
```

Comment:
yes, they have there bot repos, or at lest packages don't have a conflict. I consider to use upstream ubuntu...